implementations.md

```diff
@@ -2,75 +2,83 @@
+* Items call: http://35.160.175.42:8081/items/?limit=10 
+* Can see a particular Landsat item at: http://35.160.175.42:8081/items/LC08_L1TP_040032_20170415_20170501_01_T1
 
-Element84 made a [clojure implementation](https://github.com/Element84/catalog-api-spec/tree/dev/implementations/e84) of the spec
-that serves as a lightweight proxy to NASA's CMR, constrained to just data in USGS EROS. 
+### Other implementations
 
-Josh Fix's [open-catalog](https://github.com/joshfix/open-catalog) demonstrates generating code from the core swagger spec.
+* [Geocatalogo](https://github.com/go-spatial/geocatalogo) is a Go implementation of STAC.
 
-### Proto-software
+## Early prototypes / proof of concept
 
 Information about the software implementations that have been informing the evolution of the specification can be found at
 https://github.com/radiantearth/catalog-implementor-survey
 
-### Proto-catalogs
-[Open Imagery Network](https://openimagerynetwork.github.io/) was the first attempt at a static catalog. The OpenAerialMap 
-bucket is in active use. It is planned to evolve OIN and OAM architectures to use static catalogs.
-
-[AWS Public Datasets](http://aws.amazon.com/public-datasets/), particularly [Landsat](http://aws.amazon.com/public-datasets/landsat/), 
-[NAIP](https://aws.amazon.com/public-datasets/naip/) and [Sentinel](http://sentinel-pds.s3-website.eu-central-1.amazonaws.com/) all
-get close to the ideas of STAC static catalogs, and hopefully will evolve to implement the standard.
+* [Catalog Crawler](https://github.com/radiantearth/community-sprints/tree/master/10252017-boulder-co/catalog-crawler) was done during the [Boulder Sprint](https://github.com/radiantearth/community-sprints/tree/master/10252017-boulder-co), as an early proof of concept.
+* [py-stac](https://github.com/raster-foundry/pystac) was used to create the ISERV catalog. It is still pretty tied to that data type.
+* [go-stac](https://github.com/planetlabs/go-stac) does validation of static catalogs.
+* [Open Imagery Network](https://openimagerynetwork.github.io/) was the first attempt at a static catalog. The OpenAerialMap 
+  bucket is in active use. It is planned to evolve OIN and OAM architectures to use static catalogs.
+* [AWS Public Datasets](http://aws.amazon.com/public-datasets/), particularly [Landsat](http://aws.amazon.com/public-datasets/landsat/), 
+  [NAIP](https://aws.amazon.com/public-datasets/naip/) and [Sentinel](http://sentinel-pds.s3-website.eu-central-1.amazonaws.com/) all
+  get close to the ideas of STAC static catalogs, and hopefully will evolve to implement the standard.
+* Josh Fix's [open-catalog](https://github.com/joshfix/open-catalog) demonstrates generating code from the core swagger spec.
+* Element84 made a [clojure implementation](https://github.com/Element84/catalog-api-spec/tree/dev/implementations/e84) of the spec that serves as a lightweight proxy to NASA's CMR, constrained to just data in USGS EROS. 
```

Comment:
@m-mohr the software that is used to update the CBERS STAC catalog as new scenes are ingested is available from https://github.com/fredliporace/cbers-2-stac. There is also a static STAC crawler in https://github.com/fredliporace/stac-crawler, which I'm using to populate sat-api from a static catalog. I may prepare a text using the same structure above covering both tools.

Reply:
I just merged two files here and did not really update links or checked for up-to-dateness. But feel free to suggest updates.

Reply:
Added to the list (see chapter Ecosystem).